### PR TITLE
Issue #IQ-149 feat: sync tool changes for csp migration

### PR DESCRIPTION
**Pull request by krgauraw**

### CODE REVIEW

File: ansible/roles/setup-kafka/defaults/main.yml [hunk 1]
```diff
@@ -135,6 +135,9 @@ processing_kafka_topics:
   - name: dialcode.context.job.request.failed
     num_of_partitions: 1
     replication_factor: 1
+  - name: csp.migration.job.request
+      num_of_partitions: 1
```
Comment (vinukumar-vs) [inline ansible/roles/setup-kafka/defaults/main.yml:139]: indentation has to fix

File: ansible/roles/setup-kafka/defaults/main.yml [hunk 2]
```diff
@@ -263,4 +266,7 @@ processing_kafka_overriden_topics:
     replication_factor: 1
   - name: dialcode.context.job.request.failed
     retention_time: 1209600000
-    replication_factor: 1
\ No newline at end of file
+    replication_factor: 1
+  - name: csp.migration.job.request
+      retention_time: 1209600000
```
Comment (vinukumar-vs) [inline ansible/roles/setup-kafka/defaults/main.yml:271]: indentation has to fix

File: ansible/roles/setup-kafka/defaults/main.yml (hunk 1 above)
Comment (krgauraw) [reply ansible/roles/setup-kafka/defaults/main.yml]: done

File: ansible/roles/setup-kafka/defaults/main.yml (hunk 2 above)
Comment (krgauraw) [reply ansible/roles/setup-kafka/defaults/main.yml]: done

File: platform-tools/spikes/sync-tool/src/main/java/org/sunbird/sync/tool/mgr/CSPMigrationMessageGenerator.java [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+						nodes = util.getNodes(graphId, objectType.trim(), mimeTypeList, statusList, start, batchSize);
+					} catch (ResourceNotFoundException e) {
+						System.out.println("Error while fetching neo4j records for objectType=" + objectType + ", start=" + start + ",batchSize=" + batchSize);
+						start += batchSize;
+						continue;
+					}
+					if (CollectionUtils.isNotEmpty(nodes)) {
+						start += batchSize;
+						Map<String, String> events = generateMigrationEvent(nodes, errors);
+						sendEvent(events, errors);
```
Comment (maheshkumargangula) [inline platform-tools/spikes/sync-tool/src/main/java/org/sunbird/sync/tool/mgr/CSPMigrationMessageGenerator.java:98]: It is better to push all the events after generating in a batch.

**Review by maheshkumargangula — APPROVED** [review]

**Pull request merged**